components/mediation/tasks/org.wso2.micro.integrator.mediation.ntask/src/main/java/org/wso2/micro/integrator/mediation/ntask/NTaskTaskManager.java

```diff
@@ -574,8 +573,7 @@ public boolean isTaskDeactivated(String taskName) {
                 return false;
             }
             try {
-                return taskManager.getTaskState(taskName)
-                                  .equals(org.wso2.micro.integrator.ntask.core.TaskManager.TaskState.PAUSED);
+                return taskManager.isDeactivated(taskName);
```

Comment:
shouldn't this be isPaused()

Reply:
This method checks whether the task is deactivated or not. For standalone tasks it is whether only paused for others we have to check other conditions as well. Hence I named it as isDeactivated.